app/controllers/games_controller.rb

```diff
@@ -1,10 +1,26 @@
 class GamesController < ApplicationController
-  def new
-  end
+  def new; end
 
-  def create
-  end
+  def create; end
 
   def show
+    draw_board
```

Comment:
I like to only set instance variables once. It makes following their flow easier.

So I'd do it like this.

`@board = draw_board`

Then in your method below, replace `@board`, with `board`.

And at the end of the method, return `board`.